src/NServiceBus.Core/Transports/Learning/LearningTransportSubscriptionManager.cs

```diff
@@ -19,7 +19,7 @@ public async Task Subscribe(Type eventType, ContextBag context)
         {
             var eventDir = GetEventDirectory(eventType);
 
-            // that way we can detect that there is indeed a publisher for the event. That said it also means that we will have do "retries" here due to race condition.
+            // the subscription directory and the subscription information will be created no matter if there's a publisher for the event
```

Comment:
Nitpick: Perhaps add “... assuming that the publisher haven’t started yet” to the end?

Reply:
Amended. Can you take another look @andreasohlund ?